### PR TITLE
don't bind render on each tick

### PR DESCRIPTION
**Pull request by ngokevin**

### CODE REVIEW

File: src/core/scene/a-scene.js [hunk 1]
```diff
@@ -45,13 +45,14 @@ module.exports = registerElement('a-scene', {
 
     createdCallback: {
       value: function () {
-        this.isMobile = isMobile;
+        this.init();
         this.isIOS = isIOS;
+        this.isMobile = isMobile;
         this.isScene = true;
         this.object3D = new THREE.Scene();
+        this.renderBinded = this.render.bind(this);
```
Comment (dmarcos) [inline src/core/scene/a-scene.js]: we can just do `this.render = this.render.bind(this)` as we do in other modules

File: src/core/scene/a-scene.js [hunk 2]
```diff
@@ -45,13 +45,14 @@ module.exports = registerElement('a-scene', {
...
         this.systems = {};
         this.time = 0;
-        this.init();
```
Comment (dmarcos) [inline src/core/scene/a-scene.js]: I would kip init at the bottom. There's logic that might rely on the variables set above. For instance the logic that initializes systems might want to check for `isIOS` or `isMobile`

File: src/core/scene/a-scene.js (hunk 1 above)
Comment (ngokevin) [inline src/core/scene/a-scene.js]: I get `a-scene.js:405 Uncaught TypeError: Cannot read property 'render' of undefined`

File: src/core/scene/a-scene.js (hunk 2 above)
Comment (ngokevin) [inline src/core/scene/a-scene.js]: good call

File: src/core/scene/a-scene.js (hunk 1 above)
Comment (dmarcos) [inline src/core/scene/a-scene.js]: Do you have to set ` writable: true` in the `render` method?

Comment (ngokevin) [inline src/core/scene/a-scene.js]: ok thanks that works

**Comment by ngokevin** [comment]
Will fix the Codecov Travis failures soon. Need to remove the constraint that all PRs must improve coverage.

**Pull request merged**